.pre-commit-config.yaml

```diff
@@ -129,7 +129,6 @@ repos:
   # - id: python-check-blanket-type-ignore
   - id: python-check-mock-methods
   - id: python-use-type-annotations
-  - id: rst-backticks
```

Comment:
This pre-commit hook mistakes a link in Sphinx
```rst
`Link title <link>`_ 
```
To be a "code" block 
```rst
``inlint_code``
```
And reports a false negative.

Reply:
I cannot reproduce this locally if I add `rst-backticks` back. Can you post the error?

Reply:
Hmmm you are right. I think I made some mistakes. I try using this hook in the latest commit and it works fine. I will revert the changes.

Reply:
Reverted in https://github.com/oracle-samples/macaron/pull/232/commits/11976b13ad452d9a8c94e09cecb9aeca5c34e004